server/src/concepts/offer/model/OfferValidation.js

```diff
@@ -0,0 +1,29 @@
+import Joi from "joi-oid";
+
+const validateCreateOffer = (offer) => {
+  const schema = Joi.object({
+    title: Joi.string().min(3).max(50).required(),
+
+    description: Joi.string().max(500).required(),
+
+    city: Joi.string().max(58).required(),
+
+    phoneNumber: Joi.number().min(9).max(9).required(),
+
+    url: Joi.string().required(),
+
+    isMainPhoto: Joi.boolean().required(),
```

Comment:
we would have an array of photos like this: 
photos: [{
  url: string,
  isMainPhoto: boolean
}]